### PR TITLE
BUG in new tests

### PR DESCRIPTION
**Pull request by miguelgfierro**
### Related Issues

Failed run: https://github.com/recommenders-team/recommenders/actions/runs/6239152244

### Checklist:

- [ ] I have followed the [contribution guidelines](../CONTRIBUTING.md) and code style for this project.
- [ ] I have added tests covering my contributions.
- [ ] I have updated the documentation accordingly.
- [ ] This PR is being made to `staging branch` and not to `main branch`.

### CODE REVIEW

**Comment by miguelgfierro** [comment]
Spark nightly pass: https://github.com/recommenders-team/recommenders/actions/runs/6240331957/job/16940218200
GPU nightly pass: https://github.com/recommenders-team/recommenders/actions/runs/6239152235
CPU nightly pass:

**Review by anargyri — APPROVED** [review]

**Comment by anargyri** [comment]
We still have some references to Microsoft in the text we should review in another PR e.g. https://github.com/recommenders-team/recommenders/blob/bug/new_tests/SECURITY.md#:~:text=Security-,Microsoft,-takes%20the%20security

**Comment by anargyri** [comment]
Also in [setup.py](https://github.com/recommenders-team/recommenders/blob/05e64bd1f0ae88f40dbc56847504dab1353804fe/setup.py#L108) the Microsoft references need to be fixed.
And another PR to regenerate the documentations page.

**Comment by anargyri** [comment]
I guess also contributing license is not relevant any more and should be replaced with something from LF.

**Comment by miguelgfierro** [comment]
I updated your comments @anargyri. I´m not sure about the new email given by the LF, I´ve tried to write them to see how it works.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -68,8 +68,3 @@ Try to be empathic.
 
 </details>
 
-## Microsoft Contributor License Agreement
```
Comment (anargyri) [inline CONTRIBUTING.md:71]: @ibrahimhaddad Does LF require some similar type of agreement for contributors?

File: setup.py [hunk 2]
```diff
@@ -105,9 +105,9 @@
     url="https://github.com/recommenders-team/recommenders",
     project_urls={
         "Documentation": "https://microsoft-recommenders.readthedocs.io/en/stable/",
-        "Wiki": "https://github.com/microsoft/recommenders/wiki",
+        "Wiki": "https://github.com/recommenders-team/recommenders/wiki",
     },
-    author="RecoDev Team at Microsoft",
+    author="Recommenders contributors",
     author_email="RecoDevTeam@service.microsoft.com",
```
Comment (anargyri) [inline setup.py:111]: Email too is outdated.

File: SECURITY.md [hunk 3]
```diff
@@ -1,35 +0,0 @@
-<!-- BEGIN MICROSOFT SECURITY.MD V0.0.1 BLOCK -->
```
Comment (anargyri) [inline SECURITY.md:1]: Also @ibrahimhaddad do we need to include anything about LF secruity policy?

File: setup.py (hunk 2 above)
Comment (miguelgfierro) [reply setup.py:111]: I have sent an email to Ibrahim to see which one we can add

**Comment by loomlike** [comment]
> We still have some references to Microsoft in the text we should review in another PR e.g. https://github.com/recommenders-team/recommenders/blob/bug/new_tests/SECURITY.md#:~:text=Security-,Microsoft,-takes%20the%20security

can you create an issue for that to track? or did you already created one?

File: tests/functional/examples/test_notebooks_python.py [hunk 4]
```diff
@@ -261,6 +261,7 @@ def test_geoimc_functional(notebooks, output_notebook, kernel_name, expected_val
 
 @pytest.mark.notebooks
 @pytest.mark.experimental
+@pytest.mark.skip(reason="xLearn pip package has installation incompatibilities")
```
Comment (loomlike) [inline tests/functional/examples/test_notebooks_python.py:264]: is this temporal? or do we want to skip only at some condition, like using `skipif("some_module" not in sys.modules, reason="'some_module' was not installed correctly maybe due to version conflict")`?

**Review by loomlike — APPROVED** [review]
Thanks. I left a comment regarding skipping a test for your reference.

File: tests/functional/examples/test_notebooks_python.py (hunk 4 above)
Comment (miguelgfierro) [reply tests/functional/examples/test_notebooks_python.py:264]: In reality, it is a matter than someone has time to integrate xlearn in the core libraries

File: CONTRIBUTING.md (hunk 1 above)
Comment (miguelgfierro) [reply CONTRIBUTING.md:71]: Tracked here: https://github.com/recommenders-team/recommenders/issues/1997

File: setup.py (hunk 2 above)
Comment (miguelgfierro) [reply setup.py:111]: Tracked here: https://github.com/recommenders-team/recommenders/issues/1997

File: SECURITY.md (hunk 3 above)
Comment (miguelgfierro) [reply SECURITY.md:1]: Tracked here: https://github.com/recommenders-team/recommenders/issues/1997

**Comment by miguelgfierro** [comment]
> can you create an issue for that to track? or did you already created one?

Tracked here: https://github.com/recommenders-team/recommenders/issues/1997

**Pull request merged**